Ansible Python Test Failures

**Describe the bug**
```
set -e
  if [ -n "$GRAPHIANT_HOST" ] && [ -n "$GRAPHIANT_USERNAME" ] && [ -n "$GRAPHIANT_PASSWORD" ]; then
    echo "✅ GRAPHIANT credentials detected - running Python tests"
    python tests/test.py
  else
    echo "ℹ️  Skipping Python tests - GRAPHIANT credentials not configured"
    echo "   This is expected behavior when credentials are not set in GitHub secrets/variables"
    echo "   To enable the test, configure the following secrets/variables in GitHub:"
    echo "   - GRAPHIANT_HOST (e.g., https://api.graphiant.com)/"
    echo "   - GRAPHIANT_USERNAME"
    echo "   - GRAPHIANT_PASSWORD"
  fi
  shell: /usr/bin/bash -e {0}
  env:
    FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: true
    pythonLocation: /opt/hostedtoolcache/Python/3.12.13/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.12.13/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.13/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.13/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.13/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.12.13/x64/lib
    PYTHONPATH: /home/runner/work/graphiant-playbooks/graphiant-playbooks/ansible_collections/graphiant/naas/plugins/module_utils
    GRAPHIANT_HOST: ***
    GRAPHIANT_USERNAME: ***
    GRAPHIANT_PASSWORD: ***
✅ GRAPHIANT credentials detected - running Python tests
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.13/x64/lib/python3.12/unittest/case.py", line 403, in __init__
    testMethod = getattr(self, methodName)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'TestGraphiantPlaybooks' object has no attribute 'test_configure_backbone_site_info'. Did you mean: 'test_configure_backbone_wan_circuits'?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/graphiant-playbooks/graphiant-playbooks/ansible_collections/graphiant/naas/tests/test.py", line 1394, in <module>
    suite.addTest(TestGraphiantPlaybooks('test_configure_backbone_site_info'))
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.13/x64/lib/python3.12/unittest/case.py", line 408, in __init__
    raise ValueError("no such test method in %s: %s" %
ValueError: no such test method in <class '__main__.TestGraphiantPlaybooks'>: test_configure_backbone_site_info
Error: Process completed with exit code 1.
```

https://github.com/Graphiant-Inc/graphiant-playbooks/actions/runs/26543480639/job/78211934252

**To Reproduce**
https://github.com/Graphiant-Inc/graphiant-playbooks/actions/runs/26543480639

**Expected behavior**
https://github.com/Graphiant-Inc/graphiant-playbooks/actions/runs/26494675348

--- a/ansible_collections/graphiant/naas/tests/test.py
+++ b/ansible_collections/graphiant/naas/tests/test.py
@@ -1391,7 +1391,6 @@ if __name__ == '__main__':
     # Backbone (Core) Configuration Management Tests
     suite.addTest(TestGraphiantPlaybooks('test_configure_backbone'))
     suite.addTest(TestGraphiantPlaybooks('test_deconfigure_backbone'))
-    suite.addTest(TestGraphiantPlaybooks('test_configure_backbone_site_info'))
     suite.addTest(TestGraphiantPlaybooks('test_configure_backbone_core_to_core_interfaces'))
     suite.addTest(TestGraphiantPlaybooks('test_deconfigure_backbone_core_to_core_interfaces'))
     suite.addTest(TestGraphiantPlaybooks('test_configure_backbone_core_to_core_tunnels'))

